Ingestion/templates/datahub-executor.ecs.template.yaml

```diff
@@ -0,0 +1,668 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::LanguageExtensions
+Metadata:
+    AWS::CloudFormation::Interface:
+      ParameterGroups:
+        - Label:
+            default: Network
+          Parameters:
+            - VPCID
+            - SubnetID
+        - Label:
+            default: AWS Environment
+          Parameters:
+            - AwsRegion
+        - Label:
+            default: DataHub Info
+          Parameters:
+            - DataHubBaseUrl
+            - ExecutorId
+            - ExistingDataHubAccessTokenSecretArn
+            - DataHubAccessToken
+            - DatahubIngestionsMaxWorkers
```

Comment:
naming: we always should use DataHub as opposed to Datahub. "H" always capitalized

Reply:
renamed